src/client/sandbox/fetch.js

```diff
@@ -53,7 +55,24 @@ export default class FetchSandbox extends SandboxBase {
     }
 
     static _isValidRequestArgs (args) {
-        return typeof args[0] === 'string' || isFetchRequest(args[0]);
+        if (typeof args[0] === 'string' || isFetchRequest(args[0]))
```

Comment:
Bad design. `_isValidRequestArgs` should not modify the arguments.
So `isValidRequestArgs` should be rewritten as
```js
return typeof args[0] === 'string' || isFetchRequest(args[0]) || objectWithToStringMethod;
```

Move argument processing logic for `null`, `undefined` and custom object with `toString` method to  `FetchSandbox._processArguments(args)`.